Gemfile

```diff
@@ -1,5 +1,7 @@
 source 'https://rubygems.org'
 
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
```

Comment:
This is a standard way of fixing a security issue with the `github:` option not being secure.